hasjob/templates/detail.html.jinja2

```diff
@@ -447,7 +447,14 @@
       if (!triggered_related) {
         triggered_related = true;
         $.get("{{ post.url_for('related_posts') }}", function (data) {
-            $('#stickie-area').html(data);
+            $('#stickie-area').html(data.template);
+            window.Hasjob.Config.MaxFunnelStat = {
+              'max_impressions': data.max_impressions,
+              'max_views': data.max_views,
+              'max_opens': data.max_opens,
+              'max_applied': data.max_applied
```

Comment:
These numbers will not change on pagination. You should do this only if you're revising the colours for all stickies, not just the new ones.

Reply:
This is for the related job posts